scripts/auto_mesh.py

```diff
@@ -301,15 +303,23 @@ def query():
         # Keep only pods created after current pod
         si = ip_list.index(ip)
         ip_list = ip_list[:si]
-        return [{"role": "inter-router", "host": "config-sync-ignore-%s" % host} for host in ip_list]
+        return [{"role": "inter-router", "host": "%s" % host} for host in ip_list]
     return ip_list
 
 
 def infer():
     service_name = os.environ.get("QDROUTERD_AUTO_MESH_SERVICE_NAME", "%s-headless" % os.environ.get("APPLICATION_NAME", "amq-interconnect"))
     namespace = retrieve_namespace()
     (prefix, index) = os.environ["HOSTNAME"].rsplit("-", 1)
-    return [{"role": "inter-router", "host": "%s-%s-%s.%s.%s.svc.cluster.local" % ("config-sync-ignore-", prefix, i, service_name, namespace)} for i in range(int(index))]
+    return [{"role": "inter-router", "host": "%s-%s.%s.%s.svc.cluster.local" % (prefix, i, service_name, namespace)} for
+            i in range(int(index))]
+
+
+def generate_connector_name(connector):
+    host = connector.get("host")
+    port = connector.get("port")
+    name = "config-sync-ignore-connector/{}:{}".format(host, port)
```

Comment:
I'd suggest auto-mesh-connector or similar for the name as it is more directly tied to the purpose of the connector.

Reply:
I agree, done!